Apps/unbound/scripts/test.sh

```
@@ -202,15 +201,27 @@ else
 
-# Test 12: Verify Unbound process is running as correct user
+# Test 12: Verify Unbound process is running as correct user (CRITICAL SECURITY CHECK)
 echo -e "${YELLOW}Test 12: Checking Unbound process user...${NC}"
-PROCESS_USER=$(docker exec "$CONTAINER_NAME" ps aux 2>/dev/null | grep "[u]nbound" | head -1 | awk '{print $1}' || echo "unknown")
-if [ "$PROCESS_USER" = "unbound" ] || [ "$PROCESS_USER" = "101" ]; then
-    print_test 0 "Unbound process running as user: $PROCESS_USER"
-elif [ "$PROCESS_USER" = "root" ] || [ "$PROCESS_USER" = "0" ]; then
-    print_test 1 "Unbound process running as root (security issue)"
+# Use /proc to check the UID since ps may not be available in slim images
+PROCESS_UID=$(docker exec "$CONTAINER_NAME" cat /proc/1/status 2>/dev/null | grep "^Uid:" | awk '{print $2}' || echo "unknown")
+PROCESS_NAME=$(docker exec "$CONTAINER_NAME" cat /proc/1/status 2>/dev/null | grep "^Name:" | awk '{print $2}' || echo "unknown")
```

Shellcheck (0.11.0)

[warning] PROCESS_NAME appears unused. Verify use (or export if used externally).

(SC2034)